a_long_cat_is_long.py

```diff
@@ -0,0 +1,19 @@
+# function that receives text and returns a dictionary of length of each word
+
+
+def count_words(text):
+    newText = [letter for letter in text if letter.isalpha() or letter == ' ']
```

Comment:
Python is snake case -
https://www.google.com/amp/s/www.theserverside.com/answer/Camel-case-vs-snake-case-Whats-the-difference%3famp=1